front_end/src/components/payment/BookingInfo.js

```diff
@@ -1,62 +1,262 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
+import { withStyles, withWidth } from "@material-ui/core";
 
 import Paper from "@material-ui/core/Paper";
 import { connect } from "react-redux";
-import moment from "moment";
 import Card from "@material-ui/core/Card";
 import CardContent from "@material-ui/core/CardContent";
 import "./PaymentPage.css";
+import Table from "@material-ui/core/Table";
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Button from "@material-ui/core/Button";
+import IconButton from "@material-ui/core/IconButton";
+import InfoOutline from "@material-ui/icons/Info";
+import Checkbox from "@material-ui/core/Checkbox";
+
+const styles = theme => ({
+  cellRoot: {
+    paddingTop: 0,
+    paddingBottom: 0,
+    paddingLeft: 0,
+    paddingRight: 0
+  },
+  tableNoBorder: {
+    maxHeight: 10,
+    border: 0
+  },
+  tableBlackBorder: {
+    maxHeight: 10,
+    borderColor: "#000000"
+  },
+  disabledCheckbox: {
+    display: "none"
+  }
+});
+
+class BookingInfo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false,
+      checkedA: false,
```

Comment:
What does checkedA mean?

Reply:
ok availability...

Reply:
I can rename it. Its actually the checkbox for applying rewards points :D